README.md

```diff
@@ -1,64 +1,51 @@
-# Tower CLI app
+## Tower CLI
 
-Proof of concept of Tower command line client app based on Java native compilation, Micronaut HTTP declarative client and Tower domain classes
+Tower command line client allows to manage all the resources available at Tower. 
 
-## Installation
+### Installation
 
-From the last Github action at artifacts section, download the binary for your OS. Unzip it, give it execution permissions with `chmod +x ./tw` and move it into a folder that it's in your path (ex: `sudo mv ./tw /usr/local/bin/tw`)
+Tower CLI it is distributed as a single binary exacutable file. You only need to download your platform binary and execute it.
 
-## Build binary development version
+1. Download the latest release binary from [here](https://github.com/seqeralabs/tower-cli/releases)
+2. Enable execution permissions (`chmod +x ./tw` on Mac and Linux platforms)
+3. (OPTIONAL) Move it into a folder that is in your path.
 
-1. Download GraalVM (Java 11 version) from [this link](https://github.com/graalvm/graalvm-ce-builds/releases/tag/vm-20.2.0).
+### Quick start
 
-2. Install `native-image` tool:
+Before running the CLI:
+- You need to create a **personal access token** at Tower. See [here](https://help.tower.nf/api/overview/#authentication).
 
-    ```
-    gu install native-image
-    ``` 
-
-3. Create the native client:
-
-    ```
-     ./gradlew nativeImage
-    ```
-
-4. then run
-
-    ```
-    ./build/graal/tw
-    ```
-
-## Run non-binary development version
-
-You can run a non-binary development version running the script `./tw` at the root of this repository.
-
-## Configuration
+Running the CLI using your token:
+```
+./tw -t <your personal access token> ...
+```
 
-To use the command line you need to define these environment variables (they also can be provided using command line options):
+### Tips
 
-1. `TOWER_ACCESS_TOKEN`: (mandatory) the user access token
-2. `TOWER_WORKSPACE_ID`: (optional) the workspace id. Defaults to user workspace.
-3. `TOWER_API_ENDPOINT`: (optional) the Tower API URL. Defaults to `api.tower.nf` API.
+- Alternative you can define the `TOWER_ACCESS_TOKEN` environment variable, and then you don't need to pass the personal access token as an option.
```

Comment:
To prevent passing your personal access token at every execution it is possible to define a `TOWER_ACCESS_TOKEN` environment variable